src/mds/MDLog.cc

```diff
@@ -1316,6 +1332,9 @@ void MDLog::_replay_thread()
 
   // loop
   int r = 0;
+  start_time = ceph::coarse_mono_clock::now();
```

Comment:
Move this to when the journal is opened to avoid races with two threads updating this variable.

Reply:
done